addons/product/report/product_pricelist_templates.xml

```diff
@@ -45,7 +45,7 @@
                                         [<span t-esc="product.code"/>]
                                     </t> 
                                     <span t-esc="product.name"/>
-                                    <span t-foreach="product.attribute_value_ids" t-as="attribute_value">
+                                    <span t-foreach="product.product_template_attribute_value_ids" t-as="attribute_value">
```

Comment:
Not sure we want to see inactive attribute values here (maybe add an `_only_active()`)

Reply:
I think `ptav` from the relation `product_template_attribute_value_ids` on `product.product` should always be shown because they are what defines the variant, it's pretty much the reason why this active stuff was introduced.

Also in this case I'm not sure where `categories_data` is coming from, but if `active_test=False` is not set, no archived variant will be shown there, and the active variants will not have any inactive `ptav` anyway (unless someone messed up his database).